tests/api2/test_enable_disable_services.py

```diff
@@ -0,0 +1,10 @@
+from middlewared.test.integration.utils import call
+
+def enable_services():
+    for svc in filter(lambda x: not x['enable'], call('service.query'):
+        call('service.update', svc['id'], {'enable': True})
+
+
+def disable_services():
+    for svc in filter(lambda x: x['enable'], call('service.query'):
+        call('service.update', svc['id'], {'enable': True})
```

Comment:
Presumably `disable_services` should be:
```
call('service.update', svc['id'], {'enable': False})
```

Reply:
fixed, thanks!